server/public/html/index.html

```diff
@@ -1,239 +1,123 @@
+            <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse justify-content-center" id="navbarSupportedContent">
+            <ul class="navbar-nav">
+
+                <!-- <li class="nav-item active">
+            <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+          </li> -->
+                <li class="nav-item">
+                    <div class="input-group mb-3 form-inline my-2 my-lg-0">
+                        <a class="btn btn-default btn-outline-success" href="./help.html" target="_blank">Help</a>
+                    </div>
+                </li>
+                <li class="nav-item">
+                    <div class="input-group mb-3 form-inline my-2 my-lg-0">
+                        <a class="btn btn-default btn-outline-secondary" href="./annotatrix.html" target="_blank">Annotate!</a>
+                    </div>
+                </li>
+                <li class="nav-item">
+                    <div class="input-group mb-3 form-inline my-2 my-lg-0">
+                        <input type="text" id="gitlink" class="form-control" placeholder="URL">
+                        <div class="input-group-append">
+                            <button class="btn btn-default btn-outline-secondary" id="webload" type="submit">Load</button>
+                        </div>
+                    </div>
+
+                    <!-- <form class="form-inline my-2 my-lg-0">
+                    <input class="form-control mr-sm-2" type="search" placeholder="URL" aria-label="Search">
+                    <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Load</button>
+                </form> -->
+                </li>
+                <li class="nav-item">
+                    <div class="input-group">
+                        <div class="input-group-prepend">
+                            <!-- <span class="input-group-text" id="inputGroupFileAddon01">Upload</span> -->
+                            <button id="upload-to-server" class="btn btn-outline-secondary" type="button">Upload</button>
+                        </div>
+                        <div class="custom-file">
+                            <input type="file" class="custom-file-input" id="fileupload" aria-describedby="fileupload">
+                            <label class="custom-file-label" for="inputGroupFile01">Select local file</label>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <!-- end snippet -->
+    <div class="container">
+        <div id="welcome-content">
+
+            <!-- <h1 id="welcome-banner">Welcome to UD Annotatrix</h1> -->
+
+            <!-- <form method="post" enctype="multipart/form-data" action="./upload">
+        <input type="file" name="file" />
+        <input type="submit" value="Upload to server" />
+    </form> -->
```

Comment:
What happened to the formatting of this file?  Especially these commented-out blocks?

It was better with consistent tabs than your reformatting with spaces.  (Personally I prefer tabs.)

Could you revert the state of this file if it's just reformatting?

Reply:
I autoformatted everything in code editor. Actually, I deal with ejs only, html is autogenerated.
It's right,commented blocks were not properly indented. All I can do now is to reformat them in ejs, so in next commits they would be indented.